src/main/java/org/jenkinsci/plugins/badge/BadgeAction.java

```diff
@@ -5,36 +5,59 @@
 import jenkins.model.Jenkins;
 
 import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.QueryParameter;
-
 
 /**
 * @author Kohsuke Kawaguchi
 */
 public class BadgeAction implements Action {
+    /**
+     *
+     */
     private final BadgeActionFactory factory;
 
+    /**
+     *
+     */
     public final Job project;
-
+    /**
+     *
+     * @param factory
+     * @param project
+     */
     public BadgeAction(BadgeActionFactory factory, Job project) {
         this.factory = factory;
         this.project = project;
     }
-
+    /**
+     *
+     * @return
+     */
+    @Override
     public String getIconFileName() {
         return Jenkins.RESOURCE_PATH+"/plugin/embeddable-badges/images/24x24/shield.png";
     }
 
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.